packages/next-app/src/app/internalmanager/page.tsx

```diff
@@ -54,6 +56,13 @@ const NOTIFICATION_MAP = {
   },
 };
 
+interface IToken {
```

Comment:
Do we need IToken and ITokenInfo in `packages/next-app/src/utils/getTokenInfo.ts`?

Reply:
this was removed with the new subgraph implementation